snmp/datadog_checks/snmp/data/default_profiles/dell-powerconnect.yaml

```diff
@@ -0,0 +1,182 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+#  - dell.yaml  # TODO: check hierarchy first
```

Comment:
is this going to be addressed in a separate PR ?

Reply:
yes it's gonna be another jira ticket

Reply:
I'm wondering if we should get the hierarchy right first before implementing/merging the per product profiles 🤔

Reply:
@AlexandreYang it can be done right now of course, though I'm not sure if it's necessary. I thought it'd be better and easier to work solely on profiles and then, when everything is done, switch to this hierarchy stuff, to separate the work

Reply:
if we think this can be done, better do everything now than go back and change this in the futur